projects/demo/src/modules/app/stackblitz/starter/stackblitz-edit-button.component.ts

```diff
@@ -4,13 +4,15 @@ import {ChangeDetectionStrategy, Component} from '@angular/core';
     selector: 'stackblitz-edit-button',
     template: `
         <button
-            appearance="flat"
-            icon="assets/icons/stackblitz.svg"
+            appearance="none"
             size="s"
             tuiButton
             type="button"
         >
-            Edit
+            <a tuiLink>
+                <tui-svg src="assets/icons/stackblitz.svg"></tui-svg>
+                Edit
+            </a>
         </button>
```

Comment:
anchor inside button ? 🤔 

Should we just use the following code?
```html
<a
    icon="assets/icons/stackblitz.svg"
    iconAlign="left"
    tuiLink
>
    Edit
</a>
```

Reply:
fixed

Reply:
It shouldn't be `a`, it should be `<button tuiLink`, the link component can be applied to buttons too. `a` is a link while `button tuiLink` is a button that looks like a link. You cannot focus `a` without `href` and it is semantically wrong — what's why we made it apply to buttons too :)

Reply:
fixed